webshare/share-unicode-strings-nonutf8-manual.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="iso-8859-1">
+    <title>WebShare Manual Test: Share with non-ASCII Unicode strings in a Latin-1-encoded page</title>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <script src="resources/manual-helper.js"></script>
+  </head>
+  <body>
+    <script>
+        setup({explicit_timeout: true});
+
+        // Exact same test as in share-unicode-strings-manual.html, with same
```

Comment:
Done.

Also changed the page from `iso-8859-1` to `windows-1252`. TIL that at least on the Web platform, [the latter is the canonical name for the former](https://encoding.spec.whatwg.org/#names-and-labels).